scripts/set-default-contact/index.js

```diff
@@ -0,0 +1,117 @@
+/* eslint-disable no-console */
```

Comment:
You could technically add this one to the `.eslintrc` since it's specific to this script but either way is fine!

Reply:
thanks! noted, but I'll resolve with "either way is fine" ;)